src/App.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tory495\PhpAlgorithm;
+
+use Tory495\PhpAlgorithm\Services\MatrixService;
+
+class App
+{
+    public function run()
+    {
+        $matrixService = new MatrixService();
+        echo $matrixService->sumAdjacent(0, 0) . PHP_EOL; // 84
+        echo $matrixService->sumAdjacent(3, 2) . PHP_EOL; // 46
```

Comment:
Вместо этого лучше сделать так:
```suggestion
        echo $matrixService->sumAdjacent(0, 0) . PHP_EOL; // 84
```